OCR Action Sequencer Main.ahk

```diff
@@ -9,12 +9,12 @@ SetWorkingDir %A_ScriptDir%  ; Ensures a consistent starting directory.
 
```

Comment:
The changes to this file are irrelevant. Honestly fine with it for now, eventually a specific run instance won't show up in the base code.